umake/frameworks/games.py

```diff
@@ -210,3 +210,42 @@ def post_install(self):
                         exec='"{}" %f'.format(self.exec_path),
                         comment=self.description,
                         categories="Development;IDE;"))
+
+
+class Superpowers(umake.frameworks.baseinstaller.BaseInstaller):
+
+    def __init__(self, category):
+        super().__init__(name="Superpowers", description=_("The HTML5 2D+3D game maker"),
+                         category=category, only_on_archs=['i386', 'amd64'],
+                         download_page="https://github.com/superpowers/superpowers/releases/latest",
+                         dir_to_decompress_in_tarball='superpowers*',
+                         desktop_filename="superpowers.desktop",
+                         required_files_path=["Superpowers"])
+
+    def parse_download_link(self, line, in_download):
+        """Parse Superpowers download links"""
+        url = None
+        if "-linux-" in line:
+            in_download = True
+
+        if in_download:
+            regexp = r'href="(.*-x64.zip)"'
```

Comment:
Good one! Not a biggie to me, but it's a reasonable one (we make the same assumption in other frameworks). I like your suggestion :)